app/workers/ropensci/airtable_worker.rb

```diff
@@ -118,6 +120,19 @@ def slack_invites
       end
     end
 
+    def clear_assignments
+      params.reviewers.each do |reviewer|
+        reviewer = user_login(reviewer.to_s)
+
+        # Delete current assignment
+        reviewer_entry = airtable_revs.all(filter: "{github} = '#{reviewer}'").first
+        if reviewer_entry
+          reviewer_entry["current_assignment"] = ""
```

Comment:
Just thinking this might create a problem if someone has two current assignments but this should not happen, though. (i.e. too rare of an edge case to handle)